acceptance/tests/aix/aix_package_provider.rb

```diff
@@ -22,6 +18,11 @@ def assert_package_version(package, expected_version)
 version1 = '1.7.10.4'
 version2 = '1.8.6.4'
 
+teardown do
+  on hosts, "rm -rf #{dir}"
+  on hosts, puppet('resource', 'package', "'#{package}' ensure=absent")
```

Comment:
And all of the arguments get mashed into one string via https://github.com/puppetlabs/beaker/blob/master/lib/beaker/command.rb#L120